### PR TITLE
Add Original Field To `model.Messages` (1.)

### PR DESCRIPTION
**Pull request by VladislavsPerkanuks**
### Changes
- added `Original` field in `model.Messages` 
- updated `repo` to include this new field

### CODE REVIEW

File: pkg/convert/xliff12_test.go [hunk 1]
```diff
@@ -91,6 +91,10 @@ func Test_FromXliff12(t *testing.T) {
 				actual.Messages[i].Message = strings.Trim(actual.Messages[i].Message, "{}") // Remove curly braces for comparison
 			}
 
+			// TODO: for now restore the flag to the expected
```
Comment (VladislavsPerkanuks) [inline pkg/convert/xliff12_test.go:94]: Done in next PR

File: pkg/repo/mysql/message.go [hunk 2]
```diff
@@ -117,35 +118,44 @@ func (r *Repo) LoadMessages(ctx context.Context, serviceID uuid.UUID, opts commo
...
 		)
 
-		if err := rows.Scan(&msg.ID, &msg.Message, &msg.Description, &msg.Fuzzy, &lang); err != nil {
+		if err := rows.Scan(&msg.ID, &msg.Message, &msg.Description, &msg.Fuzzy, &lang, &original); err != nil {
 			return nil, fmt.Errorf("repo: scan message: %w", err)
 		}
 
-		messagesLookup[lang] = append(messagesLookup[lang], msg)
+		// Lookup message by language
+		messages, ok := messagesLookup[lang]
```
Comment (klavinsjanis) [inline pkg/repo/mysql/message.go:135]: alternative: 
`		messages, found := messagesLookup[lang]
		if found {
			messages.Messages = append(messages.Messages, msg)
		} else {
			messagesLookup[lang] = &model.Messages{
				Language: language.MustParse(lang),
				Messages: []model.Message{msg},
				Original: original,
			}
		}`

File: pkg/repo/mysql/message.go [hunk 3]
```diff
@@ -117,35 +118,44 @@ func (r *Repo) LoadMessages(ctx context.Context, serviceID uuid.UUID, opts commo
 
 	defer rows.Close()
 
-	messagesLookup := make(map[string][]model.Message)
+	messagesLookup := make(map[string]*model.Messages)
```
Comment (klavinsjanis) [inline pkg/repo/mysql/message.go:121]: What's the purpose of having pointers to model.Messages?

Comment (VladislavsPerkanuks) [reply pkg/repo/mysql/message.go:121]: That way modifying it here

https://github.com/translate-agent/translate/blob/ddd9e4757ba47001bbee9cd023d796b58b656ff2/pkg/repo/mysql/message.go#L145-L146

It also modifies in a lookup map

File: pkg/repo/mysql/message.go (hunk 2 above)
Comment (VladislavsPerkanuks) [reply pkg/repo/mysql/message.go:135]: I am trying to reduce `else` statements, not big fan of them

**Review by klavinsjanis — APPROVED** [review]

**Pull request merged**